apps/web/src/app/(private)/projects/[projectId]/versions/[commitUuid]/documents/[documentUuid]/_components/DocumentEditor/Editor/RefineModal/steps/2_SelectEvaluationResults/SelectableEvaluationResultsTable.tsx

```diff
@@ -1,5 +1,3 @@
-'use client'
```

Comment:
If a component is included in another component already declared as `use client` declaring the child component is not right.

Nextjs complains when passing functions as props. Everything works but you see the warnings
![image](https://github.com/user-attachments/assets/a59a501f-1833-428a-8b7c-f1707315097f)

Reply:
Hmm I don't see those warnings. Maybe the version of the linter?